libpod/oci_conmon_linux.go

```diff
@@ -1254,6 +1244,12 @@ func (r *ConmonOCIRuntime) sharedConmonArgs(ctr *Container, cuuid, bundlePath, p
 		args = append(args, rFlags...)
 	}
 
+	if ctr.config.SdNotifyMode == define.SdNotifyModeContainer {
```

Comment:
Don't we also have to unset NOTIFY_SOCKET to prevent runc and crun from blocking?  Not sure if I understood it correctly though.

Reply:
Well we don't want to pass these on to the OCI Runtimes, yes, since we don't want them handling it.